pkgs/development/libraries/oracle-instantclient/default.nix

```diff
@@ -42,7 +42,9 @@ in stdenv.mkDerivation rec {
   buildInputs = [ stdenv.cc.cc.lib ]
     ++ optionals (stdenv.isLinux) [ libaio ]
     ++ optional odbcSupport unixODBC;
-  nativeBuildInputs = [ autoPatchelfHook makeWrapper unzip ] ++ optional stdenv.isDarwin fixDarwinDylibNames;
+
+  nativeBuildInputs = [ autoPatchelfHook makeWrapper unzip ]
```

Comment:
Cosmetics: We can put the `autoPatchelfHook` into a `optional stdenv.isLinux` too.